xbmc/cores/RetroPlayer/RetroPlayer.cpp

```diff
@@ -69,9 +69,13 @@ bool CRetroPlayer::OpenFile(const CFileItem& file, const CPlayerOptions& options
 {
   CFileItem fileCopy(file);
 
+  std::string savestatePath;
+
   // When playing a game, set the game client that we'll use to open the game
-  // Currently this may prompt the user, the goal is to figure this out silently
-  if (!GAME::CGameUtils::FillInGameClient(fileCopy, true))
+  // This will prompt the user to select a savestate if there are any
+  // if there are no savestates or the user wants to create a new savestate
```

Comment:
nit
```suggestion
  // If there are no savestates or the user wants to create a new savestate
```

Reply:
When multiple sentences are found in a single comment block, you might want to use sentence separators (aka periods)